apps/emqx/i18n/emqx_schema_i18n.conf

```diff
@@ -656,8 +656,13 @@ mqtt 下所有的配置作为全局的默认值存在，它可以被 <code>zone<
 
     mqtt_idle_timeout {
         desc {
-            en: """After the TCP connection is established, if the MQTT CONNECT packet from the client is not received within the time specified by <code>idle_timeout</code>, the connection will be disconnected."""
-            zh: """TCP 连接建立后，如果在 <code>idle_timeout</code> 指定的时间内未收到客户端的 MQTT CONNECT 报文，则连接将被断开。"""
+            en: """After the TCP connection is established, if the MQTT CONNECT packet from the client is
+not received within the time specified by <code>idle_timeout</code>, the connection will be disconnected.
+After the CONNECT packet has been accepted by EMQX, if the connection idles for this long time,
```

Comment:
I think set to `infinity` will be a risk opening for DDOS because no idle timer will be started for this connection process.

The attacker could just keep establishing new TCP connections without sending MQTT.CONNECT packet that will easily bump memory usage in EMQX. 

If we don't want hibernation, just disable it with another config switch?

Reply:
We can set a maximum value, and translate infinity to max for backward compatibility. What would be the good max?

Reply:
maybe just leave the info in this doc, and let the user decide it. they just need to know the risk.